source-control/git.md

```diff
@@ -83,11 +83,23 @@ Create a PR topic --> master in Azure DevOps and approve using the squash merge
 
 ### Naming branches
 
-Let's use the following naming conventions for branches:
+When contributing to existing projects, look for and stick with the agreed branch naming convention. In open source projects this is typically found in contributing instructions, namely in `CONTRIBUTING.md` file.
```

Comment:
Nit: namely in -> in the

Reply:
Nit: ....found **in the** contributing instructions, often **in the** `CONTRIBUTING.md` file

Reply:
Thanks! Fixed.